.gitpod.yml

```diff
@@ -0,0 +1,4 @@
+# Commands to start on workspace startup. See https://www.gitpod.io/docs/config-gitpod-file
+tasks:
```

Comment:
Need some directions on the commands to add

Reply:
I need to test, nit 100% sure on my own

Reply:
Sure!